packages/openai-compatible/src/openai-compatible-chat-language-model.ts

```diff
@@ -153,7 +153,25 @@ export class OpenAICompatibleChatLanguageModel implements LanguageModelV1 {
         return {
           args: {
             ...baseArgs,
-            response_format: { type: 'json_object' },
+            response_format:
+              // TODO(shaper): Reconcile behavior below when a schema exists,
+              // re: `schema` vs `json_schema` keys.
+              //
+              // TogetherAI API docs: https://docs.together.ai/docs/json-mode
+              // { type: 'json_schema', schema: mode.schema }
+              //
+              // What we have below is taken from `OpenAIChatLanguageModel`.
```

Comment:
let's remove this from the openai compat provider (in general i try to avoid having todo comments in our codebase). we might need a special together provider (similar to groq) if they don't follow what openai does, or we might want to look into configuration points for openai-compat

Reply:
Done.